app/jobs/check_catalogues_job.rb

```diff
@@ -47,23 +40,36 @@ def catalogue_gather
         visit "http://salefinder.com.au/#{shop}-catalogue?qs=1,,0,0,0"
 
         #set the region of the catalogue we want to gather
-        page.find('a#header-change-region').trigger('click')
-        page.find('input#location-search').set(region)
-        page.find('div.autocomplete-suggestion strong').click
+        set_catalogue_region(region)
 
         #scrape the catalogue number information
-        page.all('a.catalogue-image img').each do |catalogue|
-          item_url = catalogue[:src]
-          start_count = "http://salefinder.com.au/images/featureimage/iphone/".length
-          end_count = item_url.length - (".jpg".length) - 1
-          catalogue_result <<  catalogue_num = item_url.slice!(start_count..end_count)
-        end
+        catalogue_result += scrape_catalogue_number()

```

Comment:
Ooops.. too much Javascript, get carried away putting () in a method (function in javascript)

Fix in PR #7